bin/lib/zopen-version

```diff
@@ -1,4 +1,36 @@
 #!/bin/sh
+ME=$(basename $0)
+
+printSyntax()
+{
+  echo "$ME prints version information for ${package}"
+  echo "Usage: $ME [OPTION] tool"
+  echo "   or: $ME [OPTION]"
+  echo "Syntax: zopen-version [OPTIONS]"
+  echo "where OPTIONS can be:"
+  echo "  --help: print this help"
+  echo "  --version: print version"
+}
+
+tool='zopen' # Default name
+package="zopen tools"
+copyright="Copyright (C) 2023 zOS Open Tools Community."
```

Comment:
```suggestion
copyright="Copyright (C) 2023 z/OS Open Tools Community."
```

Reply:
The slash was in there. I took it out because it confused help2man